cypress/private/cypress_test.bzl

```diff
@@ -4,6 +4,25 @@ load("@aspect_rules_js//js:libs.bzl", "js_binary_lib", "js_lib_helpers")
 load("@bazel_skylib//lib:dicts.bzl", "dicts")
 
 _attrs = dicts.add(js_binary_lib.attrs, {
+    "allow_sandbox": attr.bool(
```

Comment:
Docstring here is great but think it is misplaced here since `allow_sandbox` is a hidden kwargs parameter on the macro and not an attribute of the rule.

You could put this comment in the macro:

```
    if not kwargs.pop("allow_sandbox", False):
        # comment why where are doing this
        tags.append("no-sandbox")
```

or else hoist `allow_sandbox` do an explicit parameter in the macro and add it to the macro docstring

Reply:
this attr should just be removed entirely since it is not used at all right?